src/crewai/utilities/converter.py

```diff
@@ -64,7 +64,7 @@ def _create_instructor(self):
 
         inst = Instructor(
             llm=self.llm,
-            max_attemps=self.max_attemps,
+            max_attemps=self.max_attempts,
```

Comment:
I think we should check the "parameter" of the Instructor class max_attemps

Reply:
could not find reference to max_attemps anywhere else apart from tools_usage.py which has now been updated an commited to this PR